src/main/java/com/amazon/aocagent/commands/CommonOption.java

```diff
@@ -64,6 +71,19 @@ public Context buildContext() throws IOException, BaseException {
     }
     context.setAgentVersion(this.version);
 
+    if (!extraContexts.isEmpty()) {
+      extraContexts
+          .entrySet()
+          .forEach(
+              e -> {
+                if (e.getKey().equals("launchType")) {
```

Comment:
“ launchType”，can we define these magic code as a constant. Or in an enum?

Reply:
thx, updated